sharding/src/main/resources/application.conf

```diff
@@ -3,11 +3,17 @@ akka {
   extensions = ["com.romix.akka.serialization.kryo.KryoSerializationExtension$"]
 
   cluster {
+    
+    # !! IMPORTANT !!
+    # the below are provided by root-application.conf on our cloud nodes, 
+    # these values here serve only as local fallback
     seed-nodes = [
       "akka.tcp://cluster@127.0.0.1:2551",
```

Comment:
right, will do so here too for clarity. (Note though that these are overriden by the root-application.conf)